Frontend/Gmfctn/src/app/core/services/profile-service/profile.service.ts

```diff
@@ -0,0 +1,28 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable, OnDestroy } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { tap } from 'rxjs/operators';
+import { User } from 'src/app/shared/models/user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProfileService implements OnDestroy {
+  currentUser$: BehaviorSubject<User> = new BehaviorSubject(null as unknown as User);
+
+  private apiUrl = 'https://localhost:44349/';
```

Comment:
API host should be moved to the environment and imported into the services as locally you will have one API URL and in production it'd be another

Reply:
Fixed